src/main/resources/io/jenkins/plugins/pipelinegraphview/consoleview/PipelineConsoleViewAction/index.jelly

```diff
@@ -11,13 +11,26 @@
                 <j:if test="${it.buildable}">
                     <l:hasPermission permission="${it.permission}">
                         <j:set var="proxyId" value="${h.generateId()}" />
-                        <st:bind value="${it}" var="rebuildAction${proxyId}"/>
-                        <button id="pgv-rebuild" data-success-message="${%Build scheduled}"
-                                data-proxy-name="rebuildAction${proxyId}"
-                                class="jenkins-button jenkins-!-build-color">
-                            <l:icon src="symbol-play-outline plugin-ionicons-api"/>
-                            ${%Rebuild}
-                        </button>
+                        <st:bind value="${it}" var="rerunAction${proxyId}"/>
+                        <div class="jenkins-split-button">
+                            <button id="pgv-rerun" data-success-message="${%Build scheduled}"
+                                    data-proxy-name="rerunAction${proxyId}"
+                                    class="jenkins-button jenkins-!-build-color">
+                                <div class="jenkins-dropdown__item__icon">
+                                    <l:icon src="symbol-refresh-outline plugin-ionicons-api"/>
+                                </div>
+                                ${%Rerun}
+                            </button>
+                            <l:overflowButton id="pgv-rerun-overflow"
+                                icon="symbol-chevron-down"
+                                tooltip="${null}"
```

Comment:
why? its not required
```suggestion
```

Reply:
Because I copy/pasted from a lot of different references 😞